websites/tasks_test.py

```diff
@@ -0,0 +1,47 @@
+""" Tests for websites.tasks """
+import pytest
+from moto import mock_s3
+
+from websites.conftest import setup_s3, MOCK_BUCKET_NAME, TEST_OCW2HUGO_PREFIX
+from websites.tasks import import_ocw2hugo_course_paths, import_ocw2hugo_courses
+
+
+@pytest.mark.parametrize(
+    "paths", [["1-050-mechanical-engineering", "3-34-transportation-systems"], [], None]
+)
+def test_import_ocw2hugo_course_paths(mocker, paths):
+    """ mock_import_course should be called from task with correct kwargs """
+    mock_import_course = mocker.patch("websites.tasks.import_ocw2hugo_course")
+    import_ocw2hugo_course_paths(paths, MOCK_BUCKET_NAME, TEST_OCW2HUGO_PREFIX)
```

Comment:
This and the other task calls should use `delay()` to ensure we're calling the tasks via celery